### PR TITLE
update organization machine role section to be clearer

### PR DESCRIPTION
**Pull request by cyinma**
its not that clear to the reader that organization machine role (and member role) cannot create resources under the organization, updating it to make it clearer

### CODE REVIEW

File: docs/bsr/user-management.md [hunk 1]
```diff
@@ -29,13 +29,14 @@ request to be re-added with the desired role.
...
+- Have the [base resource role](#base-resource-roles) over the organization
+  existing resources, which defaults to [Write](#resource-write).
 
 ### Machine {#org-machine}
 
-- Can write to organization resources, regardless of the organization's
-  [base resource roles](#base-resource-roles).
+- Can view the organization and its members.
+- Have [Write](#resource-write) role over the organization existing resources, 
+  regardless of the organization's [base resource roles](#base-resource-roles).
```
Comment (rhbuf) [inline docs/bsr/user-management.md:39]:
```suggestion
- Has [Write](#resource-write) permissions over existing organization resources, 
  regardless of the organization's [base resource roles](#base-resource-roles).
```

File: docs/bsr/user-management.md [hunk 2]
```diff
@@ -29,13 +29,14 @@ request to be re-added with the desired role.
 ### Member {#org-member}
 
 - Can view the organization and its members.
-- Have the [Base resource role](#base-resource-roles) over the organizations
-  resources, which defaults to [Write](#resource-write).
+- Have the [base resource role](#base-resource-roles) over the organization
+  existing resources, which defaults to [Write](#resource-write).
```
Comment (rhbuf) [inline docs/bsr/user-management.md:33]:
```suggestion
- Has the [base resource permission](#base-resource-roles) on existing organization
 resources (the default is [Write](#resource-write)).
```

**Review by rhbuf — COMMENTED** [review]
some grammar adjustments, lmk if it makes sense

File: docs/bsr/user-management.md (hunk 2 above)
Comment (cyinma) [reply docs/bsr/user-management.md]: actually I want to keep the word `role` as they are derived / inherited, let me adjust this a bit

File: docs/bsr/user-management.md [hunk 3]
```diff
@@ -29,13 +29,14 @@ request to be re-added with the desired role.
 ### Member {#org-member}
 
 - Can view the organization and its members.
-- Have the [Base resource role](#base-resource-roles) over the organizations
-  resources, which defaults to [Write](#resource-write).
+- Derived the [base resource roles](#base-resource-roles) on existing organization
```
Comment (stefanvanburen) [inline docs/bsr/user-management.md:32]: Maybe "Inherits" is better than "Derived" here?

File: docs/bsr/user-management.md [hunk 4]
```diff
@@ -29,13 +29,14 @@ request to be re-added with the desired role.
...
+  resources (the default is [Write](#resource-write)).
 
 ### Machine {#org-machine}
 
-- Can write to organization resources, regardless of the organization's
-  [base resource roles](#base-resource-roles).
+- Can view the organization and its members.
+- Derived [Write](#resource-write) roles over existing organization resources,
```
Comment (stefanvanburen) [inline docs/bsr/user-management.md:38]: Is this still considered "derived" if it's "regardless of the base resource roles"? I think the previous wording is a little better, or @rhbuf's [suggestion](https://github.com/bufbuild/docs.buf.build/pull/324#discussion_r1067126174) of:

"Has [Write](#resource-write) permissions over existing organization resources,  regardless of the organization's [base resource roles](#base-resource-roles)."

**Review by stefanvanburen — COMMENTED** [review]
minor thoughts, these are tricky to convey 😄

File: docs/bsr/user-management.md (hunk 3 above)
Comment (cyinma) [reply docs/bsr/user-management.md]: updated both to `Inherits` 👍

File: docs/bsr/user-management.md (hunk 4 above)
Comment (cyinma) [reply docs/bsr/user-management.md]: I tried to explicitly saying this is related to role, as `Can write to` might be ambiguous on if they can create resource (technically create resource is kind of a write action), and `write permission` feels similar to me

**Review by stefanvanburen — APPROVED** [review]

**Pull request merged**